qualtricssurvey/qualtricssurvey.py

```diff
@@ -29,7 +38,7 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
     )
     survey_id = String(
         display_name=_('Survey ID:'),
-        default=_('Enter your survey ID here.'),
+        default='Enter your survey ID here.',
```

Comment:
There were still a few places that used translated values for `default`s
and this was causing more issues during testing.